src/inverseRobot.js

```diff
@@ -7,7 +7,20 @@
 */
 
 function inverseRobot(robot) {
-  // write code here
+  const KEYS = Object.values(robot);
+  const VALUES = Object.keys(robot);
+  const INVERSED_ROBOT = {};
+  const UNIQUE_KEYS = [ ...new Set(KEYS) ];
+
+  if (UNIQUE_KEYS.length !== VALUES.length) {
```

Comment:
We can use the `hasOwnProperty` for checking if the key is already present in the object